libs/blocks/section-metadata/section-metadata.css

```diff
@@ -68,29 +68,87 @@
   text-align: center;
 }
 
+.section.center[class*='-up'] {
+  justify-items: center;
+}
+
 .section.divider {
   border-bottom: 1px solid #d8d8d8;
 }
 
-.section.two-up .grid,
-.section.three-up .grid,
-.section.four-up .grid {
+.section.no-gap {
+  gap: 0;
+}
```

Comment:
Will this work given you have a gap set for `.section.two-up` etc. Seems like either the gap styles need more specific selectors or you remove the gap set in the -ups